wavenet.py

```diff
@@ -23,7 +23,8 @@ def __init__(self,
                  dilation_channels,
                  skip_channels,
                  quantization_channels=2**8,
-                 use_biases=False):
+                 use_biases=False,
+                 fast_generation=True):
```

Comment:
Consider make default `fast_generation=False` to avoid fail in the training.
(or pass `fast_generation=False` in train.py